tests/integration_tests/modules/test_ntp_servers.py

```diff
@@ -46,13 +46,17 @@ def test_dist_config_file_is_empty(
         (This test is skipped on all currently supported Ubuntu releases, so
         may not actually be needed any longer.)
         """
-        if class_client.execute("test -e /etc/ntp.conf.dist").failed:
-            pytest.skip("/etc/ntp.conf.dist does not exist")
-        dist_file = class_client.read_from_file("/etc/ntp.conf.dist")
+        if class_client.execute(
+            "test -e /etc/ntpsec/ntp.conf.dpkg-dist"
+        ).failed:
+            pytest.skip("/etc/ntpsec/ntp.conf.dpkg-dist does not exist")
+        dist_file = class_client.read_from_file(
+            "/etc/ntpsec/ntp.conf.dpkg-dist"
+        )
         assert 0 == len(dist_file.strip().splitlines())
```

Comment:
This assertion fails, the file has content. Is this due to my previous comment?

It feels like somewhere while the test instance is being customized or the last boot, `ntpsec` gets reinstalled with `/etc/ntpsec/ntp.conf` present and modified and then `etc/ntpsec/ntp.conf.dpkg-dist` gets created as a backup by dpkg.

Is this a testing error or what is the intention with this test?